consumer.go

```diff
@@ -90,7 +71,9 @@ func (c *Consumer) handlerLoop(shardID string, handler Handler) {
 		)
 
 		if err != nil {
-			log.Fatalf("Error GetRecords %v", err)
+			ctx.WithError(err).Error("GetRecords")
+			shardIterator = c.getShardIterator(shardID)
+			continue
```

Comment:
Why there's `continue` here?

Reply:
Hmm that is mess code from author. I will add more commit